api/api/utils/generate_teams_data_loader.py

```diff
@@ -40,10 +40,10 @@ def _get_students(self) -> List[Student]:
         students = self.data.get("students")
         return [
             Student(
```

Comment:
now that I think about it, couldn't we use the serializer here?

Reply:
Another PR then